editor/editor_help.cpp

```diff
@@ -2594,6 +2595,164 @@ void EditorHelpBit::_meta_clicked(String p_select) {
 	}
 }
 
+String EditorHelpBit::get_class_description(const StringName &p_class_name) const {
+	HashMap<String, DocData::ClassDoc>::ConstIterator E = EditorHelp::get_doc_data()->class_list.find(p_class_name);
+	if (!E) {
+		return String();
+	}
+
+	// Non-native properties shouldn't be translated.
+	bool is_native = ClassDB::class_exists(p_class_name);
+	// Caching descriptions would be overkill.
+	return is_native ? DTR(E->value.brief_description) : E->value.brief_description;
+}
+
+String EditorHelpBit::get_property_description(const StringName &p_class_name, const StringName &p_property_name) const {
+	HashMap<String, DocData::ClassDoc>::ConstIterator E = EditorHelp::get_doc_data()->class_list.find(p_class_name);
+	if (!E) {
+		return String();
+	}
+
+	String description;
+	if (doc_property_cache.has(p_class_name) && doc_property_cache[p_class_name].has(p_property_name)) {
+		description = doc_property_cache[p_class_name][p_property_name];
+	} else {
+		// Non-native properties shouldn't be cached, nor translated.
+		bool is_native = ClassDB::class_exists(p_class_name);
+
+		for (int i = 0; i < E->value.properties.size(); i++) {
+			String description_current = is_native ? DTR(E->value.properties[i].description) : E->value.properties[i].description;
+
+			const Vector<String> class_enum = E->value.properties[i].enumeration.split(".");
+			const String enum_name = class_enum.size() >= 2 ? class_enum[1] : "";
+			if (!enum_name.is_empty()) {
+				for (DocData::ConstantDoc val : E->value.constants) {
+					// Don't display `_MAX` enum value descriptions, as these are never exposed in the inspector.
+					if (val.enumeration == enum_name && !val.name.ends_with("_MAX")) {
+						const String enum_value = EditorPropertyNameProcessor::get_singleton()->process_name(val.name, EditorPropertyNameProcessor::STYLE_CAPITALIZED);
+						// Prettify the enum value display, so that "<ENUM NAME>_<VALUE>" becomes "Value".
+						String enum_description = is_native ? DTR(val.description) : val.description;
+						description_current = description_current.trim_prefix("\n") + vformat("\n[b]%s:[/b] %s", enum_value.trim_prefix(EditorPropertyNameProcessor::get_singleton()->process_name(enum_name, EditorPropertyNameProcessor::STYLE_CAPITALIZED) + " "), enum_description.is_empty() ? ("[i]" + DTR("No description available.") + "[/i]") : enum_description);
```

Comment:
I tried to make each arg here to be separated into its own line like it was originally, but `clang-format` had different plans...

Reply:
Sure, but the original one didn't need that, and I don't really like to declare variables that are only going to be used right below the declaration and no more.

Reply:
Welp, did it.